### DISCUSSION TITLE
Is it possible in exceljs to make different font color in a cell ? if yes, please provide me the documentation.

### DISCUSSION THREAD

**Discussion by 1amNIKHIL** [exceljs/exceljs#1987] [Q&A] [answered]
Suppose we have two word "hello world" in a cell A1 , i want hello in red color and world in black color. Is it possible through ExcelJs?

**Comment by skypesky** [comment, marked as answer]
@1amNIKHIL 
Try this. #1822.

**Comment by 1amNIKHIL** [comment]
thanks @skypesky
